packages/xo-web/src/xo-app/vm/tab-disks.js

```diff
@@ -494,13 +511,8 @@ class AttachDisk extends Component {
                 {_('vbdAttach')}
               </ActionButton>
             </span>
-            {!this._checkSr() && (
-              <div>
-                <span className='text-danger'>
-                  <Icon icon='alarm' /> {_('warningVdiSr')}
-                </span>
-              </div>
-            )}
+            {this._checkVdi() && Warning(_('warningAttachedVdi'))}
+            {!this._checkSr() && Warning(_('warningVdiSr'))}
```

Comment:
Better use a stateless component. No need to call a function on each render.
`<Warning message={} />`